releng/org.eclipse.january.releng.target/org.eclipse.january.releng.target.tpd

```diff
@@ -14,47 +14,17 @@ target "January" with source requirements
 // Using Orbit Oxygen S build at the moment. The Orbit S build contains some items
 // that the most recent R build does not (e.g. org.apache.commons.math3).
 
-location "http://download.eclipse.org/tools/orbit/downloads/drops/S20160916162009/repository" eclipse-orbit-oxygen-m2 {
+location "http://download.eclipse.org/tools/orbit/downloads/drops/S20170120205402/repository" eclipse-orbit-oxygen-m5 {
 
 // The versions we point to in Orbit are the versions we have CQs approved for, please
 // raise CQ for items not in this list or version changes
 	org.slf4j.api [1.7.2,1.7.3) // CQ 11781
 	org.apache.commons.lang3 [3.1.0,3.1.1) // CQ 11782
 	org.apache.commons.lang [2.6.0,2.6.1) // CQ 11783
-	org.apache.commons.math [2.1.0,2.1.1) // CQ 11784
 	org.apache.commons.math3 [3.5.0,3.5.1) // CQ 11785
-	ca.odell.glazedlists [1.9.0,1.9.1) // CQ 11786
-	org.antlr.runtime [4.3.0,4.3.1) // CQ 11787
-	com.google.inject [3.0.0,3.0.1) // CQ 11788
-	org.apache.log4j [1.2.15,1.2.16) // CQ 11789
 
 	// Items below are not depended on by released January bundles, only test ones, so no CQs
 	org.junit [4.12.0,5.0.0)
 	org.mockito [1.9.5,2.0.0)
 	org.hamcrest [1.1.0,2.0.0)
 }
-
-location "http://download.eclipse.org/releases/neon/" {
```

Comment:
I have gone a bit overboard with removing dependencies. Equinox is needed to run the tests from tycho because tycho runs the tests as Plug-in tests. 

The other alternative is to run the unit tests in the test phase without tycho, but not sure I want to spend time investigating that change at the moment.

(As an aside, a futher option is to ditch tycho altogether and use maven+bndtools to make the bundles as we don't need to have eclipse dependency at all, except for signing the jars)